x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/slm/SnapshotLifecycleTask.java

```diff
@@ -220,10 +223,11 @@ public ClusterState execute(ClusterState currentState) throws Exception {
 
             if (exception.isPresent()) {
                 stats.snapshotFailed(policyName);
-                newPolicyMetadata.setLastFailure(new SnapshotInvocationRecord(snapshotName, timestamp, exceptionToString()));
+                newPolicyMetadata.setLastFailure(new SnapshotInvocationRecord(snapshotName, snapshotStartTime, snapshotFinishTime,
```

Comment:
Given that the `snapshotStartTime` will always be the same as the `snapshotFinishTime` due to the constructor call on line 187, should we just pass `null` in for the start time here so that it's not saved, seeing as it doesn't help us in any way?

Reply:
Yeah I was not sure what the best way to handle this was. But this makes sense. Done.